docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
```

Comment:
This GUID is already used by another C++ topic.
You can generate a GUID any way you like. I use the Create GUID extension (nwallace.createguid)

Reply:
I generated a new one using the extension, please let me know if that one is also already taken.